test/e2e/packagemanifest_e2e_test.go

```diff
@@ -195,6 +198,28 @@ var _ = Describe("Package Manifest API lists available Operators from Catalog So
 				"quay.io/coreos/etcd-operator@sha256:c0301e4686c3ed4206e370b42de5a3bd2229b9fb4906cf85f3f30650424abec2",
 			}), "Expected images to exist in the related images list\n")
 		})
+
+		It("updating CatalogSource", func() {
```

Comment:
This could be written as : 
```
When ("the display name for catalog source is updated", func() {
       BeforeEach(func() {
            // code to update field
       }) 
       It("should successfully update the CatalogSource field", func () {
          // assertion for catsrc update validation
       })
})
```
Note: The reason for this change is going back to what our `It` spec should be. It should only have the assertions which forms the crux of the functionality being tested. Everything else lives outside.
In this case, assertions for fetching `packageManifest` & `catsrc` are part of pre-setup and hence they can live in parent container blocks (so that they get reused for future `It` specs if they use the same pre-setup)